components/ErrorBanner/ErrorBanner.tsx

```diff
@@ -1,10 +1,15 @@
-import { EthersErrorLink } from "components";
-import { mobileAndUnder } from "constant";
+import { EthersErrorLink } from "components/EthersErrorLink/EthersErrorLink";
```

Comment:
this component is the same, I just moved the styles to a separate file so I could re-use them in the new old designated voting warning banner